esp/scm/ws_account.ecm

```diff
@@ -67,7 +67,8 @@ ESPresponse [exceptions_inline] VerifyUserResponse
 	int retcode;
 };
 
-ESPservice [requiredaccess("NONE"), version("1.01"), default_client_version("1.01"), exceptions_inline("./smc_xslt/exceptions.xslt")] ws_account
+//Kevin/russ does this method need feature level check?
+ESPservice [autofeatureauth("NONE"), version("1.01"), default_client_version("1.01"), exceptions_inline("./smc_xslt/exceptions.xslt")] ws_account
```

Comment:
The ws_account is for the user who logged in. So, we do not need feature level auth.